content/concepts/payment-system-basics/accounts/reserves.md

```diff
@@ -13,22 +13,27 @@ The XRP Ledger applies _reserve requirements_, in XRP, to protect the shared glo
 
 To have an account, an address must hold a minimum amount of XRP in the shared global ledger. You cannot send this XRP to other addresses. To fund a new address, you must send that address enough XRP to meet the reserve requirement.
 
-The current minimum reserve requirement is **10 XRP**. (This is the cost of an address that owns no other objects in the ledger.) Each new [account](accounts.html) must set aside this much XRP. Some of this XRP can be recovered by [deleting the account](accounts.html#deletion-of-accounts).
-
-The [Fee Voting](fee-voting.html) process can change the reserve requirement if enough validators agree to new reserve settings.
+The reserve requirement changes from time to time due to the [Fee Voting](fee-voting.html) process, where validators can agree to new reserve settings. On [Mainnet](parallel-networks.html), the current minimum reserve requirement is **10 XRP**. (This is the cost of an address that owns no other objects in the ledger.) Each new [account](accounts.html) must set aside this much XRP. Some of this XRP can be recovered by [deleting the account](accounts.html#deletion-of-accounts).
 
+To determine the current minimum reserve requirement, use the [`server_info` command](server_info.html) and take `validated_ledger.reserve_base_xrp`.
 
 ## Base Reserve and Owner Reserve
 
-The reserve requirement is divided into two parts:
+The reserve requirement is has two parts:
 
-* The **Base Reserve** is a minimum amount of XRP that is required for each address in the ledger. Currently, this is 10 XRP (`10000000` [drops](xrp.html#xrp-properties)).
-* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger. Currently, this is 2 XRP (`2000000` [drops](xrp.html#xrp-properties)) per item.
+* The **Base Reserve** is a minimum amount of XRP that is required for each address in the ledger.
+* The **Owner Reserve** is an increase to the reserve requirement for each object that the address owns in the ledger.
```

Comment:
I'm not a fan of moving the key numbers out of the bullet points. I think putting them into the paragraphs in their respective sections makes them easier to miss.

Reply:
For developers, they should not be considered key numbers, since they can change (and have changed multiple times already).